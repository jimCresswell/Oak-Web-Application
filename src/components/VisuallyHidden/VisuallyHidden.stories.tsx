import React from "react";
import { ComponentStory, ComponentMeta } from "@storybook/react";

import Component from "./VisuallyHidden";

export default {
  title: "Accessibility/VisuallyHidden",
  component: Component,
  argTypes: {},
} as ComponentMeta<typeof Component>;

const Template: ComponentStory<typeof Component> = (args) => (
  <div>
    <Component {...args} />
  </div>
);

export const VisuallyHiddenExample = Template.bind({});

VisuallyHiddenExample.args = {};
