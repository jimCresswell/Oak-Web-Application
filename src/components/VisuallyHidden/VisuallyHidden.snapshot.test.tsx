import { render } from "@testing-library/react";

import VisuallyHidden from "./VisuallyHidden";

it("renders <VisuallyHidden> unchanged", () => {
  const { container } = render(<VisuallyHidden>Some text</VisuallyHidden>);
  expect(container).toMatchSnapshot();
});
