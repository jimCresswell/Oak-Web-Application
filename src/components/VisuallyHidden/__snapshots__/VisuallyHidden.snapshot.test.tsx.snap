// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <VisuallyHidden> unchanged 1`] = `
<div>
  <div
    class="visuallyHidden"
  >
    Some text
  </div>
</div>
`;
